apps/ledger-live-desktop/src/renderer/components/TransactionConfirm/ConfirmTitle.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useMemo } from "react";
+import { TFunction } from "i18next";
+import { withTranslation } from "react-i18next";
+import {
+  Account,
+  AccountLike,
+  SubAccount,
+  TransactionCommon,
+  TransactionStatusCommon,
+} from "@ledgerhq/types-live";
+import { Transaction, TransactionStatus } from "@ledgerhq/live-common/generated/types";
+import Text from "~/renderer/components/Text";
+
+type Props = {
+  t: TFunction;
+  Title:
+    | React.ComponentType<{
+        account: Account | SubAccount;
+        parentAccount: Account | null | undefined;
+        transaction: TransactionCommon;
+        status: TransactionStatusCommon;
+      }>
+    | undefined;
+  account: AccountLike;
+  parentAccount: Account | undefined | null;
+  status: TransactionStatus;
+  transaction: Transaction;
+  typeTransaction: string | undefined;
+};
+
+const ConfirmTitle = ({
+  t,
+  Title,
+  account,
+  parentAccount,
+  status,
+  transaction,
+  typeTransaction,
+}: Props) => {
+  const description: string = useMemo(() => {
+    if (typeTransaction === "Approve") return t("approve.description");
+    else return t("sign.description");
+  }, [t, typeTransaction]);
+
+  if (Title) {
+    return (
+      <Title
+        account={account}
+        parentAccount={parentAccount}
+        transaction={transaction}
+        status={status}
+      />
+    );
+  }
```

Comment:
By expecting a `ReactNode` instead we can simplify this part and do it like we would with children (we could also use children here) and we can also remove the other props we don't need now
```suggestion
  if (title) {
    return title;
  }
```